main.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"golang.org/x/mod/modfile"
+	"golang.org/x/mod/semver"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	nvmInstallMessage = "You can download the required Node.js version using Node Version Manager (https://github.com/nvm-sh/nvm). Once nvm is installed, run `nvm install` in this directory to install the correct Node version."
+)
+
+var mmplugCmd = &cobra.Command{
+	Use:   "mmplug",
+	Short: "mmplug is a command line tool to manage plugin projects",
+}
+
+var doctorCmd = &cobra.Command{
+	Use:   "doctor",
+	Short: "Check if your development environment set up correctly",
+	Run:   runDoctor,
```

Comment:
Have you tried RunE? you can link a function that returns an err. Not 100% sure but I guess that would handle stderr and os.Exit in a better way (if you look at mmctl repo it's always handled like that).